app/ui-message/client/message.js

```diff
@@ -177,6 +177,14 @@ Template.message.helpers({
 			return msg.avatar.replace(/^@/, '');
 		}
 	},
+	customAvatarUsername() {
```

Comment:
```suggestion
	getUserAvatar() {
	... here you can rely on the roomType.getAvatarPath()...
```

Reply:
hi @renatobecker This is not working. It gives all users the same avatar